rest-api/src/main/java/gov/cms/qpp/conversion/api/services/QrdaServiceImpl.java

```diff
@@ -39,6 +46,15 @@ public ConversionReport convertQrda3ToQpp(Source source) {
 		return converter.getReport();
 	}
 
+	/**
+	 * Retrieve the CPC+ validation file for the QPP service
+	 *
+	 * @return cpc+ validation file.
+	 */
+	public InputStream getCpcPlusValidationFile() {
+		return storageService.getCpcPlusValidationFile();
+	}
```

Comment:
Usually "get" functions are reserved for simple POJO or Collection actions. This one has a side effect of opening a stream to S3. This is more like an open action. This comment could be said for both this method definition and the storage service definition as well.

Reply:
I see. Would changing the name to `retrieveS3CpcPlusValidationFile` with some better commenting suffice?

Reply:
yeah just refactor this

Reply:
It's been refactored.